editoast/src/views/search.rs

```diff
@@ -683,6 +685,47 @@ pub(super) struct SearchResultItemScenario {
     tags: Vec<String>,
 }
 
+#[derive(Search, Serialize, ToSchema)]
+#[search(
+    table = "train_schedule",
+    column(name = "timetable_id", data_type = "integer"),
+    column(name = "train_name", data_type = "string")
+)]
+#[allow(unused)]
+/// A search result item for a query with `object = "trainschedule"`
+pub(super) struct SearchResultItemTrainSchedule {
+    #[search(sql = "train_schedule.id")]
+    id: u64,
+    #[search(sql = "train_schedule.train_name")]
+    train_name: String, // useless puisqu'on en a besoin pour la request?
```

Comment:
```suggestion
    train_name: String,
```

We comment in English 🫘 ;)

I'd suggest to leave it because 1) all other search responses also include their criteria 2) it makes the API more convenient and "stateless" to use as we don't have to "remember" the data each request was made with.